inference/core/workflows/core_steps/models/foundation/clip_comparison.py

```diff
@@ -184,4 +196,10 @@ def _post_process_result(
             prediction[ROOT_PARENT_ID_KEY] = (
                 image.workflow_root_ancestor_metadata.parent_id
             )
+            # removing fields from `inference` response model
+            # that are not registered as outputs
```

Comment:
Should we register these as outputs? Or handle context/benchmarking in a different way?

Reply:
I guess we should not, as those are mainly `Nones` in practice. Apart from that - similarly to `sv.Detections(...)` change, this problem will be gone when we build better version of step - encapsulating output into something more well designed